src/tools/tidy/src/style.rs

```diff
@@ -79,11 +79,11 @@ fn line_is_url(line: &str) -> bool {
                 => state = EXP_URL,
 
             (EXP_LINK_LABEL_OR_URL, w)
-                if w.starts_with("http://") || w.starts_with("https://")
+                if w.starts_with("http://") || w.starts_with("https://") || w.starts_with("../")
```

Comment:
There is currently one line in the codebase which would be affected:

```
$ git grep -E '\s?//(/|!)?\s\.\./'
src/libstd/net/tcp.rs:    /// ../../std/net/trait.ToSocketAddrs.html#tymethod.to_socket_addrs
```

[It is part of a Markdown link](https://github.com/rust-lang/rust/blob/c523b3f95498cf35a2f1a6bb8c071394ca4a6143/src/libstd/net/tcp.rs#L118-L119).